src/Traits/UserHasTeams.php

```diff
@@ -112,10 +114,9 @@ public function isOwnerOfTeam($team)
     {
         $team_id = $this->retrieveTeamId($team);
 
-        return ($this->teams()
+        return (bool) $this->teams
             ->where('owner_id', $this->getKey())
-            ->where('team_id', $team_id)->first()
-        ) ? true : false;
+            ->where('id', $team_id)->first();
```

Comment:
```suggestion
            ->where('id', $team_id)->exists();
```

Reply:
hi this suggestions is very recommendable

Reply:
thank you for suggestions

Reply:
But your suggestion is not suitable for this reason:

Method Illuminate\Database\Eloquent\Collection::exists does not exist.

$this->teams return a collection object.